eng/tools/lint-diff/src/markdown-utils.ts

```diff
@@ -0,0 +1,221 @@
+import * as commonmark from "commonmark";
```

Comment:
Migrate to `.github/src` (after this PR)

Reply:
Added to https://github.com/Azure/azure-rest-api-specs/issues/33302